CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+* **footers:** change 'Terms & Conditions' to 'Terms and conditions' ([#1564](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1564)) ([9e3c9ba](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/9e3c9bafd1b9d54ffbe3ad8907b448983a6d5430))
+* **forms:** add validation icons for `<select>`s and fix its position for `<textarea>` ([#1371](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1371)) ([ceeff1d](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/ceeff1d2bbcb21b4ca66a067b876ec10b56bee84))
+* **forms:** prevent that valid icon hides input content ([#1306](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1306)) ([a64f2a1](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/a64f2a185ee2ca9ff497fe2194c822daf41536a6))
+* **icons:** use the right close icon ([#1566](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1566)) ([82da281](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/82da28120b7304b99887060c87ce109b650897fb))
+* live preview URL in PR template ([#1546](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1546)) ([1777b02](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/1777b02efa0f11db3bcb5a4c419170ef386453bd))
+* **modal:** remove unused CSS variables ([#1418](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1418)) ([1251579](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/125157936c1da4cdac6446438cea413b9a518908))
+* **navbar:** minor fixes ([#1498](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1498)) ([5bd74eb](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/5bd74eb043b137efc25d771ee7d3a8233a629947))
+* **pagination:** remove CSS variables ([#1457](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1457)) ([92c1052](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/92c10527b036921fa3f27021f84e1dd986916d8a))
+* **paginations:** add missing breakpoints after lg ([#1579](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1579)) ([6255243](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/625524333d29c25fcb2fa4020b4e2c5c5c839fb7))
+* **stepped process:** improve rendering of arrow shapes ([#1485](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1485)) ([308a9f1](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/308a9f1d03e213b6ae2e3673c89d49c7b6c11231))
+* **tags:** handle close button rendering issues ([#1574](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1574)) ([7b3c494](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/7b3c49493b5d71b0e7355c2214c0d767a034b3ed))
```

Comment:
Do we need this kind of commit ?

Reply:
This file is generated automatically and I don't touch it. In the future, I'd prefer to use directly the content we create in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1429 or just have a list of the versions and the links to the releases that contain the precise release notes.